x/bank/client/cli/tx.go

```diff
@@ -143,3 +149,51 @@ When using '--dry-run' a key name cannot be used, only a bech32 address.
 
 	return cmd
 }
+
+//TODO: change msgfees related docs
+func GetUpdateDenomMetadataProposal() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "update-denom-metadata-proposal <path-to-proposal-file.json> <deposit>",
+		Aliases: []string{"udmp", "u-d-m-p"},
+		Args:    cobra.ExactArgs(2),
+		Short:   "Proposal to update fee denom metadata",
+		Long:    strings.TrimSpace(`Submit a proposal to update fee denom metadata along with an initial deposit.`),
+		Example: fmt.Sprintf(`$ %[1]s tx msgfees update-denom-metadata-proposal path/to/proposal/file.json 1000000000nhash
+$ %[1]s tx msgfees udmp path/to/proposal/file.json 1000000000nhash
+$ %[1]s tx msgfees u-d-m-p path/to/proposal/file.json 1000000000nhash
```

Comment:
Should be `tx bank` here instead of `tx msgfees`. But if it gets moved to the gov stuff, that part will change again to something like `tx gov submit-legacy-proposal`.